components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/dto/OAuthAppRevocationRequestDTO.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
update the license header

Reply:
Addressed in [ec021ae](https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/1627/commits/ec021ae9bac9964777efc58182dd369d6f054e52)